### PR TITLE
Move netperf image to k8s-staging-perf-tests

### PR DESCRIPTION
**Pull request by marseel**
Move netperf image to k8s-staging-perf-tests.
I've already pushed the image to this image repository.

#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

### CODE REVIEW

File: network/benchmarks/netperf/launch.go [hunk 1]
```diff
@@ -74,7 +74,7 @@ func init() {
 	flag.IntVar(&iterations, "iterations", 1,
 		"Number of iterations to run")
 	flag.StringVar(&tag, "tag", runUUID, "CSV file suffix")
-	flag.StringVar(&netperfImage, "image", "sirot/netperf-latest", "Docker image used to run the network tests")
+	flag.StringVar(&netperfImage, "image", "gcr.io/k8s-staging-perf-tests/netperf", "Docker image used to run the network tests")
```
Comment (aojea) [inline network/benchmarks/netperf/launch.go:77]:
```suggestion
	flag.StringVar(&netperfImage, "image", "gcr.io/k8s-staging-perf-tests/netperf:v0.0.1", "Docker image used to run the network tests")
```

It seems this image is tagged

Comment (marseel) [reply network/benchmarks/netperf/launch.go:77]: hmm, I've added tagging image as `latest` too, so we don't need to specify tag here/update it each time we push new image.

Comment (aojea) [reply network/benchmarks/netperf/launch.go:77]: I see, confused the code changes

Comment (aojea) [reply network/benchmarks/netperf/launch.go:77]: I saw you added versuon v0.0.1 above

**Pull request closed without merging**